src/app/app.utils.js

```diff
@@ -17,4 +17,28 @@ function acornPlural(amount) {
 
```

Comment:
You should never pollute the global scope with your own functions, unless you're making a helper library, where you would define an access variable. Better idea is to have these helper utils in an `angular.factory` and inject them where necessary. This makes it easier to test them and provides clear separation.